artifacts/attributes.adoc

```diff
@@ -12,7 +12,7 @@
 :product-very-short: RHDH
 :product-version: 1.4
 :product-bundle-version: 1.4.0
-:product-chart-version: 1.4.0
+:product-chart-version: 1.4
```

Comment:
> It impacts:
> 
>  grep -ire product-chart-version modules/
> modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:podman pull registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-chart-version}
> modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc:podman push --remove-signatures registry.redhat.io/rhdh/rhdh-hub-rhel9:{product-chart-version} default-route-openshift-image-registry.<hostname>/<project_name>/rhdh-hub-rhel9:{product-chart-version}
> modules/installation/proc-install-rhdh-ocp-helm-cli.adoc:helm upgrade redhat-developer-hub -i https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz
> modules/installation/proc-install-rhdh-ocp-helm-cli.adoc:helm upgrade redhat-developer-hub -i "https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz" \
> modules/installation/proc-rhdh-deploy-aks-helm.adoc:helm -n $NAMESPACE install -f values.yaml $DEPLOYMENT_NAME openshift-helm-charts/redhat-developer-hub --version {product-chart-version}
> modules/installation/proc-rhdh-deploy-aks-helm.adoc:helm upgrade $DEPLOYMENT_NAME -i "https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz" \
> modules/installation/proc-rhdh-deploy-aks-helm.adoc:helm upgrade $DEPLOYMENT_NAME -i https://github.com/openshift-helm-charts/charts/releases/download/redhat-redhat-developer-hub-{product-chart-version}/redhat-developer-hub-{product-chart-version}.tgz
> modules/installation/proc-rhdh-deploy-eks-helm.adoc:  [--version {product-chart-version}] \
> modules/installation/proc-rhdh-deploy-gke-helm.adoc:  --version {product-chart-version}
> modules/upgrade/proc-upgrade-rhdh-helm.adoc:  openshift-helm-charts/redhat-developer-hub --version {product-chart-version}
> modules/configuring-external-databases/proc-configuring-postgresql-instance-using-helm.adoc:helm upgrade -n <your-namespace> <your-deploy-name> openshift-helm-charts/redhat-developer-hub -f values.yaml --version {product-chart-version}

The problem is that the Chart version should remain 1.4.0 in the other docs, but the RHDH image used in the Helm airgap procedure should either be 1.4 or 1.4-1734106454 or the exact digest used in that Chart version (`registry.redhat.io/rhdh/rhdh-hub-rhel9@sha256:48edcf6f736e17f33d3630ce2fddc19e95316b7824a7af24e9f0df48ac4f4fe3` from what I can see).